Morphic/Morphic.tex

```diff
@@ -559,6 +559,31 @@ \subsection{Keyboard events}
 To discover the key values, you can open a Transcript window and add \glbind{Transcript} \ct{show: anEvent keyValue} to \mthref{handleKeystroke}.
 The \ct{anEvent} argument of \ct{handleKeystroke:} is an instance of \clsind{KeyboardEvent}, another subclass of \clsind{MorphicEvent}. Browse this class to learn more about keyboard events.
 
+\subsection{Global accessors}
+
+In complex applications, one may need to refer to the currently processed event without having it available in the current method.
+While we should avoid such situations in general, there is an accessor message defined on \ct{Object}, \mthind{Object}{currentEvent}, that allows us to retrieve the event currently being handled again from any method in any object.
+To do so, we can simply write \ct{self currentEvent} in an arbitrary method.
+Analogously, there are \mthind{Object}{currentHand} and \mthind{Object}{currentWorld} to retrieve the current \clsind{HandMorph} (which represents your cursor) or \clsind{PasteUpMorph} (which contains all other morphs displayed at your screen).
+
+In some cases, one also may want to directly control the user interaction without following the usual event handling control flow.
```

Comment:
"In some cases, one may also want to..."

Reply:
Resolved via 54bf6db8700fd02c4d21d8f3af08e68843378390.